b_cfn_elasticsearch_cloner/initial_cloner_source/action.py

```diff
@@ -0,0 +1,142 @@
+import json
+import logging
+import os
+from decimal import Decimal
+from typing import Dict, Any, Optional, Tuple
+
+import boto3
+from elasticsearch import Elasticsearch, RequestsHttpConnection
+from elasticsearch.serializer import JSONSerializer
+from elasticsearch.helpers import bulk
+from requests_aws4auth import AWS4Auth
+
+
+logger = logging.getLogger(__name__)
+
+dynamodb = boto3.resource("dynamodb")
+
+class Action:
```

Comment:
Please add a docstring to this Action class telling what this class does. Since it works only on creation event, please be explicit about that in your docstring.